spec/grape/endpoint_spec.rb

```diff
@@ -655,6 +655,38 @@ def app
     end
   end
 
+  describe '#declared; from a nested mounted endpoint - case 2' do
+    before do
+      mounted = Class.new(Grape::API) do
+        params { requires :inner_val, type: Integer }
+        get { declared(params) }
+      end
+
+      subject.format :json
+      subject.namespace :outer do
+        params { requires :outer_val, type: Integer }
```

Comment:
Well, this should be executed before `namespace :outer`, I guess it should be something like:
```ruby
      subject.format :json
      subject.params { requires :outer_val, type: Integer }
      subject.namespace :outer do
        get { declared(params) }
        namespace :inner do
          mount mounted
        end
```

And then your specs are passed locally. What do you think? /cc @dblock